integration-test/helpers/harness.js

```diff
@@ -222,7 +222,7 @@ export function withIOSContext (test) {
             testInfo.skip(testInfo.project.name !== 'webkit')
 
             const context = await browser.newContext({
-                ...devices.iPhone,
+                ...devices['iPhone 13'],
```

Comment:
This wasn't really working without specifying the iPhone version.

Reply:
Ah nice!